data/docker-apache2/letsencrypt/csr/0000_csr-certbot.pem

```diff
@@ -0,0 +1,26 @@
+-----BEGIN CERTIFICATE REQUEST-----
```

Comment:
This *must not* be committed, as it's persisted data but totally controlled by docker.